Updated automated publishing steps (#3664)
microsoft/vscode-docs: api/working-with-extensions/continuous-integration.md

## Automated Publishing

You can also configure the CI to publish a new version of the extension automatically.

The publish command is similar to publishing from a local environment using the [`vsce`](https://github.com/Microsoft/vsce) service but the command needs to also include the Personal Access Token (PAT). [`vsce`](https://github.com/Microsoft/vsce) can also pick up on the `VSCE_PAT` environmental variable for the Personal Access Token.

You shouldn't expose the PAT with the rest of the source code (it's a sensitive information), so you can store it in a "secret variable". The value of that variable will not be exposed and you can use it in the CI pipline.

# Example Implementations

The following examples use a sample project that uses [Yarn](https://yarnpkg.com/), but can be adapted to use [npm](https://www.npmjs.com/) as well.

> 

Reviewer: I suggest to simply remove this from the start and just add an FAQ item at the end of the page.

You can configure the CI to publish a new version of the extension automatically.

> The publish command is similar to publishing from a local environment using the [`vsce`](https://github.com/Microsoft/vsce) service but the command needs to also include the Personal Access Token (PAT). By default, [`vsce`](https://github.com/Microsoft/vsce) will use an environmental variable named `VSCE_PAT` (if defined) for the Personal Access Token.

Reviewer: How about this instead?

```suggestion
The publish command is similar to publishing from a local environment using the [`vsce`](https://github.com/Microsoft/vsce) service but the command needs to also include the Personal Access Token (PAT). [`vsce`](https://github.com/Microsoft/vsce) can also pick up on the `VSCE_PAT` environmental variable for the Personal Access Token.
```

You shouldn't expose the PAT with the rest of the source code (it's a sensitive information), so you can store it in a "secret variable". The value of that variable will not be exposed and you can use it in the `azure-pipelines.yml` file.

> To create a secret variable named `VSCE_PAT`, follow the [Azure DevOps Secrets instructions](https://docs.microsoft.com/azure/devops/pipelines/process/variables?tabs=classic%2Cbatch#secret-variables).

Reviewer: How about:

```suggestion
To setup `VSCE_PAT` as a secret variable, follow the [Azure DevOps Secrets instructions](https://docs.microsoft.com/azure/devops/pipelines/process/variables?tabs=classic%2Cbatch#secret-variables).
```
